broadinstitute/pearl
# JN-132  export download

This adds a minimally functional download dialog and button to the participant list
![image](https://user-images.githubusercontent.com/2800795/234418871-c649b137-a803-45a5-acd4-ea57c724b47d.png)
This also adds an enrollee limit to ExportOptions so that the preview page won't crash the server.  

TO TEST:
1. restart AdminApiApp
2. go to https://localhost:3000/ourhealth/studies/ourheart/env/sandbox/participants
3. click "Download"
4. see the dialog (all options are currently disabled -- nothing about the download is configurable yet)
5. press "Download"
6. confirm a file is downloaded like "2023-04-26-enrollees.txv"
7. Open the file in Numbers, confirm you can see the rows of data
